datasync/chaindatafetcher/kafka/kafka.go

```diff
@@ -30,12 +30,21 @@ const (
 	// item indices of message header
 	MsgHeaderTotalSegments = iota
 	MsgHeaderSegmentIdx
+	MsgHeaderVersion
+	MsgHeaderProducerId
 	MsgHeaderLength
+	LegacyMsgHeaderLength = 2
```

Comment:
Why don't you separate this value from iota constians?

Reply:
I made separate constant for `LegacyMsgHeaderLength`.